lib/ansible/plugins/lookup/env.py

```diff
@@ -29,11 +29,11 @@
 
 EXAMPLES = """
 - name: Basic usage
-  debug:
+  ansible.builtin.debug:
     msg: "'{{ lookup('env', 'HOME') }}' is the HOME environment variable."
```

Comment:
```suggestion
    msg: "'{{ lookup('ansible.builtin.env', 'HOME') }}' is the HOME environment variable."
```
To correctly use FQCNs everywhere, this also needs to be changed.